matter_server/server/device_controller.py

```diff
@@ -383,6 +383,24 @@ async def send_device_command(
                 interactionTimeoutMs=interaction_timeout_ms,
             )
 
+    @api_command(APICommand.WRITE_ATTRIBUTE)
+    async def write_attribute(
+        self,
+        node_id: int,
+        attribute_path: str | list[str],
```

Comment:
Maybe only support a single path? Is it common to write the same value to multiple paths at the same time?